site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -349,15 +338,9 @@ the steps in [Viewing Warnings][27].
 
 ## Update permissions {: #update_permissions }
 
-Updating an extension with additional permissions may temporarily disable it. The user will have to
+Updating an extension with additional permissions with warnings will temporarily disable it. The user will have to
```

Comment:
I believe the extension will only be disabled if the user hasn't already approved the warning string, but there are also subtleties here related to permission request collapsing (e.g. `"tabs"` not shown if you already have `"<all_urls>"`).

This may be a case where we can't accurately cover the nuance and should therefore encourage developers to test permissions upgrades in Chrome to see how their specific upgrade behaves.

Reply:
Thanks for clarifying! I'll revert this change.